<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    #ROS#
    <param name="frame_id" value="map" />
    <param name="scan_topic" value="scan"/>
    <param name="cloud_topic" value="cloud"/>
    <param name="scan_topic1" value="scan1"/>
    <param name="cloud_topic1" value="cloud1"/>
    #DATA#
    <param name="min_dist" value="0.01"/>
    <param name="max_dist" value="50.0"/>
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="output_cloud2" value="false"/>
    <param name="output_360" value="true" />
    <param name="reversed" value="false"/>
    <param name="hard_resample" value="false"/>
    <param name="soft_resample" value="false"/>
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="-3.1415926"/>
    <param name="max_angle" value="3.1415926"/> 
    <rosparam param="mask1" >[-3.14,3.14]</rosparam>
    <!-- <rosparam param="mask2" >[-1,0]</rosparam> -->
    #CONNECT#
    <param name="type" value="udp" />
    <param name="local_port" value="6668" />
    <param name="number" value="2" />

    <param name="lidar_ip" value="192.168.158.98"/>
    <param name="lidar_port" value="6543" />
    <param name="lidar1_ip" value="192.168.158.99"/>
    <param name="lidar1_port" value="6543" />
    #GET#
    <param name="uuid" value="-1" />
    #SET#
    <param name="rpm" value="-1"/>
    <param name="resample_res" value="-1"/>
    <param name="with_smooth" value="-1" />
    <param name="with_deshadow" value="-1" />
    <param name="alarm_msg" value="-1"/>
    <param name="direction" value="-1"/>
  </node>
 </group>
</launch>
